$NetBSD: patch-configure.ac,v 1.2 2026/05/07 16:34:35 adam Exp $

Check for symbol that is in libcurses, not in libterminfo.

--- configure.ac.orig	2026-05-05 07:19:07.000000000 +0000
+++ configure.ac
@@ -62,17 +62,17 @@ AC_ARG_WITH(
 	[with_ui=no]
 )
 AS_IF([test "x$with_ui" != xno], [
-	AC_CHECK_LIB([ncursesw],[tparm],[
+	AC_CHECK_LIB([ncursesw],[ungetch],[
 		CURSESLIB=-lncursesw
         AC_CHECK_LIB([tinfow],[tgetent], [
             CURSESLIB="$CURSESLIB -ltinfow"
         ])
-	],[AC_CHECK_LIB([curses],[tparm],[
+	],[AC_CHECK_LIB([curses],[ungetch],[
 		CURSESLIB=-lcurses
         AC_CHECK_LIB([tinfo],[tgetent], [
             CURSESLIB="$CURSESLIB -ltinfo"
         ])
-	],[AC_CHECK_LIB([ncurses],[tparm],[
+	],[AC_CHECK_LIB([ncurses],[ungetch],[
 		CURSESLIB=-lncurses
         AC_CHECK_LIB([tinfo],[tgetent], [
             CURSESLIB="$CURSESLIB -ltinfo"
@@ -106,7 +106,7 @@ AS_IF([test "x$with_readline" != xno],
 		TERMLIB=-ltinfow
     ],[AC_CHECK_LIB([tinfo],[tgetent], [
 		TERMLIB=-ltinfo
-    ],[AC_CHECK_LIB([curses],[tparm],[
+    ],[AC_CHECK_LIB([curses],[ungetch],[
 		TERMLIB=-lncurses
 	],[AC_CHECK_LIB([termcap],[tgetent],[
 		TERMLIB=-ltermcap
